dev/tasks/bundle/buildclassiceditor.js

```diff
@@ -0,0 +1,60 @@
+/**
```

Comment:
`bundle-config.js` makes sense. Passing all this in the commandline would be hard anyway. Too many things are needed there.